cpp/include/copc-lib/las/config.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef COPCLIB_LAZ_CONFIG_H_
```

Comment:
How will this relate to copc config? Should we have copc config subclass this?

Reply:
Yep we will